### ISSUE TITLE
Job 29:12b ULB those without fathers

### ISSUE THREAD

**Issue by SusanQuigley** [Door43/ulb-en#574]
Job 29:12b ULB
because I used to rescue the poor person who cried out,
and also **those without fathers,** who had no one to help **him**.
The blessing of him who was about to perish came on me;
I caused the widow's heart to sing for joy.

Job uses singular nouns to refer to groups of people in general. 
Could ULB  "those without fathers" be changed to "the fatherless" or "him without a father"?

**Comment by TomWarren** [comment]
BDB notes that it should not be implied that both parents are dead, but that he has no father.

The ULB of JOB 29:12 how reads: 
\v 12 because I rescued the one who was poor when he cried out,
\q and the one who had no father when he had no one to help him.


The UDB of JOB 29:12 now reads:
\v 12 because I had helped those who were poor when they cried out for help,
\q2 and because I aided those who had no fathers, those who had no one else to help them.

Thanks Susan, 
Look that over. 

Tom
